### DISCUSSION TITLE
Miryoku Kmonad Lite with numpad/symbols

### DISCUSSION THREAD

**Discussion by nomm-dot** [manna-harbour/miryoku#316] [Q&A] [answered]
I don't see why the Kmonad Lite layout does not have the numpad and symbols. Not too worried about the mouse layer since either you have a trackpad/point available if you're using a laptop. Additionally, I don't see why the keys adjacent to the spacebar aren't leveraged to use for layers they aren't perfectly ergonomic, but reaching them with your thumbs is completely comfortable at least for myself. Giving 3 available thumb keys for layers. And if you use num+shift(homerow J) like i do to access the symbols anyways that opens up the possibilities even more. 

Hold spc for nav layer
(I like this so I can nav with one hand)

Hold key right of spc for num, +shift for sym

Hold key left of space for fun, +shift for media
(I like Miryoku by default having fun layered with the numbers, but on a lot of keyboards media is on the same keys as function anyways)

The num and fun layers could also swap sides since I'm sure a lot of people are familiar using a numpad with their right hand anyways. 

Or even just put the lesser fun key on the home row on G and H then you could overlay the function keys on the num layer or both sides and have the media layer dedicated to the key left of space then youd even have room to +shift for a mouse layer on the right side.

**Comment by manna-harbour** [comment, marked as answer]
On a standard row-staggered keyboard, only Space is easily reachable with the thumb, so that's the only thumb key enabled in the Lite mapping.

If you happen to have a keyboard with more keys that are comfortably reachable with the thumb (e.g. a keyboard with split space, or something like a Japanese keyboard with a small spacebar and additional keys on that row), you can substitute a custom mapping with https://github.com/manna-harbour/miryoku/discussions/85.

However, if you disregard the relative positions of thumb keys you'll introduce a muscle memory incompatibility with Miryoku on other keyboards.

Also note, as mentioned in the docs, the other layers all still accessible, just not via thumb keys.

Right hand Num, Fun, etc. is included as an option: https://github.com/manna-harbour/miryoku/tree/master/docs/reference#flip.
